README.md

```diff
@@ -28,13 +28,19 @@ Be prepared to demonstrate your understanding of this week's concepts by answeri
 
 1. What is the DOM?
 
+The dom is the Document Object Model, it is the stored value of the entire contents of our page and allows us to modify the html and css without making changes to the respective files
+
 2. What is an event?
+An event is something that changes our page when the user triggers a specific command. There are many events that occur on the page such as click, hover, mouseover, etc.
 
 3. What is an event listener?
+We use an event listener in our javascript to specify the particular event which will occur on the page
```

Comment:
An event listener is a function that will run whenever the relevant event occurs on the object it is attached to.